src/service_provider/service_provider.routes.spec.js

```diff
@@ -0,0 +1,53 @@
+import request from "supertest";
+import app from "../app";
+import { FindAllServiceProviders, CreateServiceProvider, FindAllServiceProvidersById, UpdateServiceProvider, DeleteServiceProvider } from './service_provider.services'
+
+jest.mock('./service_provider.services')
+
+const fakeServiceProviders = [
+    {
+        id: 'aaabbb111',
+        name: 'test'
+    }
+]
+
+describe("Service Provider Router Tests", () => {
```

Comment:
I assume all these "return empty array" checks are placeholders?

Reply:
I updated the tests to be more clear about what they're testing.